website/docs/r/hpc_cache.html.markdown

```diff
@@ -83,7 +83,15 @@ The following arguments are supported:
 * `directory_ldap` - (Optional) A `directory_ldap` block as defined below.
 
 ~> **Note:** Only one of `directory_active_directory`, `directory_flat_file` and `directory_ldap` can be set.
- 
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `key_vault_key_id` - (Optional) Specifies the URL to a Key Vault Key.
+
+~> **NOTE:** `auto_key_rotation_enabled` must be set to `false` when updating `key_vault_key_id`.
```

Comment:
that'd be a provider behaviour we're missing, to disable this, then re-enable it during the update?

Reply:
Yes make sense, I've updated the code to trigger another update when the key is updated